### DISCUSSION TITLE
[Feb 08] long long text

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1272] [Discord Threads]
> **Erkin Tek** _queried at_ ```2022-02-07T15:19:21.412000+00:00```  
## I'm new to nocodb, just added some data to a table, one column is "long text" and it occupies  5 pages, i may hide it but can i made smaller like " long long tex... "?  
  
> **o1lab** _commented at_ ```2022-02-07T16:03:25.896000+00:00```  
#### Did you mean changing the type? If so you can click on more options. And you should be able to change to exact data type  
  
> **Erkin Tek** _commented at_ ```2022-02-07T16:39:19.803000+00:00```  
#### no, I want data to be long and there, but in the grid view a little shorter  
  
> **o1lab** _commented at_ ```2022-02-07T16:41:33.871000+00:00```  
#### Got it  
  
> **moosefist** _commented at_ ```2022-02-07T18:20:05.267000+00:00```  
#### Is there a way to only show the first couple lines, and hit a button to expand?  
  
> **o1lab** _commented at_ ```2022-02-07T18:21:12.551000+00:00```  
#### Happy to take a PR  
  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/940265487097544745)  
 
 - - - -

#### Join [NocoDB's](http://www.nocodb.com/)  community
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by Gari-sincere** [comment]
Definitely agree that this is an important feature.  I want like some of the features of NocoDB over Baserow but this is a killer for me.  Baserow has Longtext compacted by default and then expands it out when the cell is selected. But with NocoDB I'm stuck with scrolling offscreen display for long inputs.

![image](https://user-images.githubusercontent.com/44096102/201439429-6d602e27-139a-4d7d-a4b4-cc0fcc3c5cb7.png)
